Augmented docs and tests; fixed Tests; fixed _get_shortest_route() bug

gwselke opened the pull request:
Addressing https://github.com/manwar/Map-Tube/issues/19.
Suggestion for addressing (https://github.com/manwar/Map-Tube/issues/18) .
Added/augmented test cases (also added to MANIFEST).
Added documentation for publicly available maps.
Suggest bumping version to 4.03 (but only done in the two .pm files that I have changed).

manwar commented:
Great work, thanks as always. @gwselke

manwar commented:
@gwselke, I noticed after merge when I ran the test on my local box, the test just hanged on `t/ok_map_forked.t` forever. I checked the process, it was taking `100%` CPU all the time. I forced stopped the process eventually.

Is this unusual?
Or am I doing something wrong?

Please advise.

gwselke commented:
Oops? Will check tomorrow, certainly shouldn't happen! 
-- 
\Gisbert


-------- Ursprüngliche Nachricht --------
Von: Mohammad Sajid Anwar ***@***.***>
Gesendet: 22. März 2025 22:08:52 MEZ
An: manwar/Map-Tube ***@***.***>
CC: gwselke ***@***.***>, Mention ***@***.***>
Betreff: Re: [manwar/Map-Tube] Augmented docs and tests; fixed Tests; fixed _get_shortest_route() bug (PR #20)

manwar left a comment (manwar/Map-Tube#20)

@gwselke, I noticed after merge when I ran the test on my local box, the test just hanged on `t/ok_map_forked.t` forever. I checked the process, it was taking `100%` CPU all the time. I forced stopped the process eventually.

Is this unusual?
Or am I doing something wrong?

Please advise.


-- 
Reply to this email directly or view it on GitHub:
https://github.com/manwar/Map-Tube/pull/20#issuecomment-2745728792
You are receiving this because you were mentioned.

Message ID: ***@***.***>
